### PR TITLE
Add Reasonable defaults

### PR DESCRIPTION
**Pull request by povilasv**

### CODE REVIEW

File: base/local/prometheus.yaml [hunk 1]
```diff
@@ -209,4 +209,5 @@ spec:
       resources:
         requests:
           storage: 200Gi
+      storageClassName: ebs-gp2
```
Comment (george-angel) [inline base/local/prometheus.yaml]: I remember this rubbish, it gets ugly with gcp, but I don't think there is a solution to it.

File: example/thanos-compact.yaml [hunk 2]
```diff
@@ -6,4 +6,4 @@ spec:
   resources:
     requests:
       storage: 250Gi
-  storageClassName: ebs-gp2
+  storageClassName: ebs-gp3
```
Comment (george-angel) [inline example/thanos-compact.yaml]: What?

Comment (povilasv) [reply example/thanos-compact.yaml]: it's an example :D so I used to check merged config

File: base/local/prometheus.yaml (hunk 1 above)
Comment (povilasv) [reply base/local/prometheus.yaml]: To be fair we wouldn't need storage classes if we had default storage class set to ebs-gp2

https://kubernetes.io/docs/tasks/administer-cluster/change-default-storage-class/

**Pull request merged**